src/silicone/database_crunchers/quantile_rolling_windows.py

```diff
@@ -53,7 +53,10 @@ class QuantileRollingWindows(_DatabaseCruncher):
 
     With these weightings, the desired quantile of the data is then calculated. This
     calculation is done by sorting the data and interpolating the value where the
-    cumulative sum of weights equals the quantile.
+    cumulative sum of weights equals the quantile. Quantiles less than half the weight
+    of the smallest follow value, or more than 1 - half the weight of the largest follow
```

Comment:
```suggestion
    of the smallest follow value, or more than one minus half the weight of the largest follow
```

Reply:
incorporated with modification of linebreaks